tsl/src/bgw_policy/job.c

```diff
@@ -502,6 +505,8 @@ policy_refresh_cagg_read_and_validate_config(Jsonb *config, PolicyContinuousAggD
 				 errhint(
 					 "The max batches per execution should be greater than or equal to zero.")));
 
+	refresh_newest_first = policy_refresh_cagg_get_refresh_newest_first(config);
+
```

Comment:
Since this is only set if `policy_data` is defined, it makes more sense to move this to inside the if-statement below.

Reply:
Actually this `policy_refresh_cagg_read_and_validate_config` is a bit ugly. IMHO it should return `PolicyContinuousAggData` instead. But I prefer to send a following PR doing the refactoring.